src/components/App.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import Login from './Login/Login';
 
 const App = () => (
   <div>
-    Datalyzer test
+    <Switch>
+      <Route exact path="/" component={Login} />
```

Comment:
Why not to add Private route for '/' path?

Reply:
> Why not to add Private route for '/' path?

Out of ticket scope. Will be implemented with auth flow ticket (https://trello.com/c/JQfxtcj8/24-fe-implement-login-logic-using-proper-api)